<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Manejo de Neuronas</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>207</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="agregarInicio">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>405</y>
       <width>94</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar al inicio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="agregarFinal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>88</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar al final</string>
     </property>
    </widget>
    <widget class="QPushButton" name="guardar">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recuperar">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>210</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Recuperar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="buscarId">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>350</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string extracomment="Buscar neurona por id"/>
     </property>
     <property name="placeholderText">
      <string extracomment="Buscar neurona por id"/>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>380</y>
       <width>181</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sortId">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>410</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordenar por id</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortVoltaje">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>440</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordenar por voltaje</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_QScene">
    <attribute name="title">
     <string>QScene</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>581</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrarPuntos">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>518</y>
       <width>246</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar sólo ́los puntos (x, y) de las neuronas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrarLineas">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>548</y>
       <width>297</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar las líneas dibujadas a sus puntos más cercanos</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>191</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="busquedaAmplitud">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>520</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Búsqueda en amplitud</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
